pkg/common/connectionstore/connection_lookup.go

```diff
@@ -23,6 +23,8 @@ const (
 	keyPattern         = "%s_%s"
 	connIDKeyPrefix    = "conn"
 	connStateKeyPrefix = "connstate"
+	invKeyPrefix       = "inv"
+	eventDataKeyprefix = "event-data"
```

Comment:
This is probably now too generic :).

Reply:
fixed to `connevent` to match `connstate`